flocker/node/agents/cinder.py

```diff
@@ -189,59 +284,70 @@ def _is_cluster_volume(cluster_id, cinder_volume):
     return False
 
 
-def _blockdevicevolume_from_cinder_volume(cinder_volume):
+def _blockdevicevolume_from_cinder_volume(cinder_volume, host_map):
     """
     :param Volume cinder_volume: The ``cinderclient.v1.volumes.Volume`` to
         convert.
     :returns: A ``BlockDeviceVolume`` based on values found in the supplied
         cinder Volume.
     """
+    if cinder_volume.attachments:
+        # There should only be one.
```

Comment:
Cinder and Nova `Volume` instances have an *attachments* attribute which says to me that a volume can be attached to multiple hosts.
But I'm not sure if that's actually possible.
And it certainly shouldn't happen with Flocker managed volumes.